src/policy/feerate.cpp

```diff
@@ -22,7 +24,7 @@ CAmount CFeeRate::GetFee(uint32_t num_bytes) const
 {
```

Comment:
Wouldn't it make sense to rename this from `GetFee` to `RoundFee` to clarify that this is not a getter, but a calculation?

At the very least the doxygen should be updated.

Reply:
Updated doxygen. I think keeping it `GetFee` is fine.